packages/web/index.js

```diff
@@ -59,7 +59,10 @@ module.exports = (neutrino, opts = {}) => {
     options.devServer.proxy = {
       '**': {
         target: options.devServer.proxy,
-        changeOrigin: true
+        changeOrigin: true,
+        headers: {
```

Comment:
This is only set when `options.devServer.proxy === 'string'`. Should this go in the main `devServer` defaults above?

Reply:
I don't think so…this, along with the `**` property and `changeOrigin: true` are meant to be be the "easy" way of enabling a proxy.

If you wanted to specify an obj, then I think we assume you know what you're doing and you'll set your own http-proxy-middleware options.

Make sense?

Reply:
Makes sense to me, we should probably document this though.

Reply:
Indeed…thought we had. I'll add to this PR.

Reply:
Ok I added some docs for devServer.proxy